code/modules/mob/living/silicon/robot/modules/nanotrasen/medical.dm

```diff
@@ -0,0 +1,97 @@
+		/datum/prototype/robot_iconset/hover_handy/medical,
+		/datum/prototype/robot_iconset/hover_drone/surgery,
+		/datum/prototype/robot_iconset/biped_insekt/medical,
+		/datum/prototype/robot_iconset/hover_glitterfly/surgeon,
+		/datum/prototype/robot_iconset/biped_miss/medical,
+		/datum/prototype/robot_iconset/hover_x88/medical,
+		/datum/prototype/robot_iconset/grounded_mechoid/medical,
+		/datum/prototype/robot_iconset/biped_noble/medical,
+		/datum/prototype/robot_iconset/grounded_zoomba/medical,
+		/datum/prototype/robot_iconset/grounded_worm/crisis,
+		/datum/prototype/robot_iconset/raptor/medical,
+		/datum/prototype/robot_iconset/hover_coffin/medical,
+		/datum/prototype/robot_iconset/baseline_old/medical_2,
+	)
+
+/datum/prototype/robot_module/nanotrasen/medical/create_mounted_item_descriptors(list/normal_out, list/emag_out)
+	..()
+	if(normal_out)
+		normal_out |=  list(
+			/obj/item/borg/sight/hud/med,
+			/obj/item/healthanalyzer/advanced,
+			/obj/item/reagent_containers/borghypo/surgeon,
+			/obj/item/reagent_containers/glass/beaker/large,
+			/obj/item/reagent_scanner/adv,
+
+			// Surgery things.
+			/obj/item/autopsy_scanner,
+			/obj/item/surgical/scalpel/cyborg,
+			/obj/item/surgical/hemostat/cyborg,
+			/obj/item/surgical/retractor/cyborg,
+			/obj/item/surgical/cautery/cyborg,
+			/obj/item/surgical/bonegel/cyborg,
+			/obj/item/surgical/FixOVein/cyborg,
+			/obj/item/surgical/bonesetter/cyborg,
+			/obj/item/surgical/circular_saw/cyborg,
+			/obj/item/surgical/surgicaldrill/cyborg,
+			/obj/item/gripper/no_use/organ,
+
+			/obj/item/gripper/medical,
+			/obj/item/shockpaddles/robot,
+			/obj/item/reagent_containers/dropper, // Allows surgeon borg to fix necrosis
+			/obj/item/reagent_containers/syringe/unbreakable,
+			/obj/item/roller_holder,
+			/obj/item/robot_builtin/dog_mirrortool,
+		)
+	if(emag_out)
+		var/obj/item/reagent_containers/spray/acid_spray = new
+		acid_spray.reagents.add_reagent(/datum/reagent/acid/polyacid::id, 250)
+		acid_spray.name = "Polyacid spray"
+		emag_out |= acid_spray
+
+/datum/prototype/robot_module/nanotrasen/medical/provision_resource_store(datum/robot_resource_store/store)
+	..()
+	store.provisioned_stack_store[/obj/item/stack/medical/advanced/bruise_pack] = new /datum/robot_resource/provisioned/preset/bandages/advanced
+	store.provisioned_stack_store[/obj/item/stack/medical/advanced/ointment] = new /datum/robot_resource/provisioned/preset/ointment/advanced
+	store.provisioned_stack_store[/obj/item/stack/medical/splint] = new /datum/robot_resource/provisioned/preset/splints
+	store.provisioned_stack_store[/obj/item/stack/nanopaste] = new /datum/robot_resource/provisioned/preset/nanopaste
+
+// todo: this is evil
+/datum/prototype/robot_module/nanotrasen/medical/legacy_custom_regenerate_resources(mob/living/silicon/robot/robot, dt, multiplier)
+	..()
+	for(var/obj/item/reagent_containers/spray/maybe_evil_acid_spray in robot.robot_inventory.provided_items)
```

Comment:
note to reviewers:

there's no good way to do this right now without a lot of additional code
this was how it was in the past so it's kept this way

 in the future this should be changed